images/win/scripts/Installers/Install-MSYS2.ps1

```diff
@@ -0,0 +1,92 @@
+################################################################################
+##  File:  Install-MSYS2.ps1
+##  Desc:  Install MSYS2 and 64-bit gcc, cmake, & llvm (32-bit commented out)
+################################################################################
+
+# References
+# https://github.com/msys2/MINGW-packages/blob/master/azure-pipelines.yml
+# https://packages.msys2.org/group/
+
+$dash = "—"
+$wid = 80
+
+$orig_path = $env:PATH
+$git_path  = "C:\Program Files\Git"
+
+# get info from https://sourceforge.net/projects/msys2/files/Base/x86_64/
+$msy2_uri  = "http://repo.msys2.org/distrib/x86_64/msys2-base-x86_64-20190524.tar.xz"
+$msy2_sha1 = "cfe5035b1b81b43469d16bfc23be8006b9a44455".ToUpper()
+
+$msy2_file = "C:/msys2.tar.xz"
```

Comment:
We have the environment variable AGENT_TEMPDIRECTORY in the image with the path to temp files/folders. Can we use it?

Reply:
Done.  Changed to:
```
$msy2_file = "$env:AGENT_TEMPDIRECTORY/msys2.tar.xz"
```